litgpt/eval/evaluate.py

```diff
@@ -84,22 +81,22 @@ def convert_and_evaluate(
     out_dir.mkdir(parents=True, exist_ok=True)
 
     save_filepath = out_dir / Path("results.json") if save_filepath is None else Path(save_filepath)
-    config_filepath = checkpoint_dir/"model_config.yaml"
 
-    with open(config_filepath, encoding="utf-8") as f:
-        config_dict = yaml.safe_load(f)
-    repo_id = f"{config_dict['hf_config']['org']}/{config_dict['hf_config']['name']}"
-
-    copy_config_files(source_dir=checkpoint_dir, out_dir=out_dir)
-
-    model_path = out_dir / "model.pth"
+    model_path = out_dir / "pytorch_model.bin"
```

Comment:
I wonder if we should skip the saving step if a `pytorch_model.bin` file already exists (unless `force_conversion=True`). What do you think?

Reply:
Arg, I missed it. All good!